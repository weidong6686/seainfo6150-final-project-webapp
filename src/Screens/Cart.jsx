import React from 'react';
import './css/Cart.css';
import { Link } from 'react-router-dom';
import data from '../data';

const Cart = (props) => {
    const item = data.products[0];
    return (
    <div className="cart">
        <div className="cart-list">
            <ul className="cart-list-container">
                <li>
                    <h3>Shopping Cart</h3>
                </li>
                <li>Price</li>
                <li>
                <div className="cart-image">
                  <img src={item.image} alt="product" />
                </div>
                <div className="cart-name">
                  <div>
                    <Link to={"/product/" + item._id}>
                      {item.name}
                    </Link>
                  </div>
                  <div>
                    Qty: <select>
                                <option>1</option>
                                <option>2</option>
                                <option>3</option>
                                <option>4</option>
                            </select>
                    <button>
                      Delete
                    </button>
                  </div>
                </div>
                <div className="cart-price">
                  ${item.price}
                </div>
              </li>
            </ul>
        </div>
        <div className="cart-action">
            <h3>
                Subtotal ( 1 items) : $ {item.price}
            </h3>
            <Link to="/order" className="button-cart">Proceed to Checkout</Link>
        </div>

    </div>

    );
};

export default Cart;