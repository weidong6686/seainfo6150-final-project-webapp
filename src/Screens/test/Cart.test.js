import React from "react";
import Cart from "../Cart.jsx";

describe("Cart tests", () => {
  it("renders correctly", () => {
    const { container } = (<Cart />);
    expect(container).toMatchSnapshot();
  });
});