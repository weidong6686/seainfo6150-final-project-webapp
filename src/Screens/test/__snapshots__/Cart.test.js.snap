// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart tests renders correctly 1`] = `undefined`;
